### DISCUSSION TITLE
Thank you for Lychee 5 !

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [LycheeOrg/Lychee#2132] [General]
I just want to thank the project team for the new release. Lychee 5 comes in a clean design, which makes it even more my favourite photo app for the web :-)

Well, there are some things to be fixed or added, like this is the case after every major release. With this release, I tried a new host (PikaPod), as the manual upgrade from the older version was not successful yet.

I have some minor issues in web and mobile seen, I will add them later along with some ideas.

Looking forward to the further development of Lychee!

**Comment by ildyria** [comment]
Thank you for your kind words.

Our team is very small and time quite limited (family, GF, activities...) but we try our best. :)

**Comment by audioscavenger** [comment]
i came back to delete the comment but you went ahead! Just like Linuxserver.io we are all like-minded! thank you for your hard work!!

**Comment by ildyria** [comment]
We are on version 6 now, way better than version 5! :D

**Reply by audioscavenger** [reply]
> We are on version 6 now, way better than version 5! :D
Can't wait to see that
